src/main/markdown/doc/latest/tutorial/i18n.md

```diff
@@ -364,13 +366,13 @@ You tell the GWT compiler that StockWatcher now supports the German (de) locale
     *  The English language version loads by default.
 3.  Load the German version.
     *  To the end of the URL, append `&locale=de`
-    *  `http://localhost:8888/StockWatcher.html?gwt.codesvr=127.0.0.1:9997&locale=de
+    *  http://localhost:8888/StockWatcher.html?gwt.codesvr=127.0.0.1:9997&locale=de
```

Comment:
I think I'd have rather added the closing backtick. Are there other such URLs in the tutorial? How are they rendered?